### DISCUSSION TITLE
Toggle Sidebar feature

### DISCUSSION THREAD

**Discussion by devnchill** [TheOdinProject/curriculum#29206] [Ideas]
On every site there's a list of topics on right side which helps to navigate to that specific topic . It is helpful but it also pushes the main content a bit towards left . I think a toggle feature for it would be great . 
![swappy-20241219-162116](https://github.com/user-attachments/assets/c996085f-f562-405f-84b2-f41a6c92d8dd)

**Comment by mao-sz** [comment]
Looks like something for the [website repo](https://github.com/TheOdinProject/theodinproject/discussions) instead of the curriculum repo.

**Comment by devnchill** [comment]
ah sorry , i see there's already a suggestion there . I'll close this one .
